src/components/Settings/Notifications/NotificationsWebPush/index.tsx

```diff
@@ -32,6 +35,12 @@ const NotificationsWebPush: React.FC = () => {
 
   return (
     <>
+      {!window.location.protocol.startsWith('https') && (
+        <Alert
+          title={intl.formatMessage(messages.httpsRequirement)}
+          type="warning"
+        />
+      )}
```

Comment:
This will crash on server side render because window will not exist. We need to check in a useEffect which only runs in the client and set state based on the result.